pyramid/config/views.py

```diff
@@ -804,22 +816,21 @@ def wrapper(context, request):
                 stacklevel=4,
                 )
 
+        if accept is not None:
+            if is_nonstr_iter(accept):
+                raise ConfigurationError(
+                    'A list is not supported in the "accept" view predicate.',
+                )
+            accept = accept.lower()
```

Comment:
Could we instead run it through the normalizer in WebOb? This may be a stupid question...

Reply:
yep, it will be once I push changes

Reply:
Well to be clear I'm running them through `parse_offer` to check for validity but that api does not normalize. Webob doesn't expose an api right now to normalize offers so we still do `.lower()` in pyramid.

Reply:
I looked briefly into making `parse_offer` do the normalization but it was a pain to do correctly because it requires reconstructing an offer with the params, which requires re-quoting things properly. If that were added to `webob.acceptparse.AcceptOffer.__str__` then I would use it.

Reply:
last night we updated `AcceptOffer` to normalize the offers so this is fixed now and pyramid no longer does the `lower()` itself.